components/bl-multiple-bar-chart-component/src/validate.js

```diff
@@ -1,21 +1,12 @@
 export const validate = items => {
-  if (!items) {
-    return [];
-  }
-  
-  if (!items[0].hasOwnProperty('objectId')) {
-    return items.map(item => ({
-      ...item,
-      objectId: getId(),
+  if (Array.isArray(items)) {
```

Comment:
```
export const validate = items => {
  if (!Array.isArray(items)) {
    return []
  }

  return items.map(({ name, goal, progress }) => ({
    name,
    id: name,
    goal: goal > 0 ? goal : 1,
    progress: Math.max(0,progress),
  }));
};
```

That looks for me more clear and informative

we have corner case with handling:
```
  if (!Array.isArray(items)) {
    return []
  }
  ```
  
  and default behavior:
  
  ```
return items.map(({ name, goal, progress }) => ({
  name,
  id: name,
  goal: goal > 0 ? goal : 1,
  progress: Math.max(0,progress),
}));
  ```